android/app/src/main/kotlin/net/defined/mobile_nebula/MainActivity.kt

```diff
@@ -54,6 +54,7 @@ class MainActivity: FlutterActivity() {
     }
 
     override fun configureFlutterEngine(flutterEngine: FlutterEngine) {
+        super.configureFlutterEngine(flutterEngine)
```

Comment:
With this change, the call to `GeneratedPluginRegistrant.registerWith(flutterEngine)` can be removed:

https://github.com/flutter/engine/blob/d0ad7cf69bdf78ddda0956b89d2a7d0c9b26fd2c/shell/platform/android/io/flutter/embedding/android/FlutterActivity.java#L1328-L1337

Another ref: https://docs.flutter.dev/platform-integration/platform-channels#step-3-add-an-android-platform-specific-implementation